addons/chemical/functions/fnc_breathing.sqf

```diff
@@ -0,0 +1,47 @@
+#include "script_component.hpp"
+
+/*
+ * Author: DiGii
+ * 
+ * Arguments:
+ * 0: Target <OBJECT>
+ *
+ * Return Value:
+ * NONE
+ *
+ * Example:
+ * [player] call kat_chemical_fnc_breathing;
+ *
+ * Public: No
+*/
+params ["_unit"];
+
+[
+	{
+		params ["_unit"];
+		goggles _unit in KAT_AVAIL_GASMASK
+	},
+	{
+		params ["_unit"];
+		[
+		{
+			params["_args","_handler"];
+			_args params ["_unit"];
+			if !(goggles _unit in KAT_AVAIL_GASMASK || !(alive player) || _unit getVariable["ace_medical_heartrate",80] <= 0) then {
+				[_handler] call CBA_fnc_removePerFrameHandler;
+				[_unit] spawn FUNC(breathing);
+			} else {
+				if(GET_PAIN_PERCEIVED(_unit) >= 0.4 || _unit getVariable["ace_medical_heartrate",80] >= 105) then {
```

Comment:
_unit getVariable["ace_medical_heartrate",80] should be changed into _unit getVariable [QACEGVAR(medical,heartrate), 80]